src/Libraries/Microsoft.Extensions.AI/ChatCompletion/FunctionInvokingChatClient.cs

```diff
@@ -840,6 +869,19 @@ FunctionResultContent CreateFunctionResultContent(FunctionInvocationResult resul
         return result;
     }
 
+    /// <summary>This method will invoke the fuction within the try block.</summary>
```

Comment:
Is this true? I don't see any `try` block.

Reply:
Line 831 happens within a `try` block, this description can be improved.

Reply:
Oh, thanks. I interpreted the XML comment to mean that *this* method contains a try block. It's a bit odd for the XML docs here to refer to code in the caller, given that the method doesn't know who is going to call it.